CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v1.21.0
+------------------------------
+*May 29, 2019*
+
+### Updated
```

Comment:
Very minor and maybe for next time but this should be `changed` :)

Reply:
Yep – can you update (as otherwise, likely won't get done)

Reply:
There's one further down too.